### DISCUSSION TITLE
labs-open gopro upload and COHN - feature request

### DISCUSSION THREAD

**Discussion by Fougl** [gopro/labs#908] [Feature Requests]
Hi, I am proposing an amazing feature - when a recording, like motion detection is made, a new upload command could be developed, where an automatic upload to specific local IP, would be possible, where port fowarding or something similar would be used to get the video. This would be an upgrade to the current COHN capabilities of Hero 12.

**Comment by dnewman-gpsw** [comment]
Sounds like a request for the Open GoPro team as Labs doesn't have any networking controls. I don't even know what COHN is.

**Comment by Fougl** [comment]
i read in one of the discussions, that you are thinking about joining forces with open gopro.

well COHN is the capability of Hero 12 to connect to the router, and you can manipulate all gopros connected to the router through local IP assigned to them. But GoPro labs could be the addtion to that, when the recording is made it makes the upload automatically. Now I need to scan each GoPro continuously if new footage was recorder, and download the videos. Its still amazingly better than Hero 11, where this was not possible. But combining Labs and open gopro would make a monster
